commands/active.go

```diff
@@ -12,20 +12,23 @@ import (
 	"github.com/docker/machine/libmachine/state"
 )
 
-func cmdActive(c *cli.Context) {
+func cmdActive(c *cli.Context) error {
 	if len(c.Args()) > 0 {
-		fatal("Error: Too many arguments given.")
+		return errors.New("Error: Too many arguments given")
```

Comment:
Typically I like to reserve `errors.New` for named errors and then use `fmt.Errorf` to return a one-off message.

i.e.
```
var ErrTooManyArguments = errors.New("too many arguments given")
...

return ErrTooManyArguments
```